metaci/plan/migrations/0033_auto_20200121_0110.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.2.9 on 2020-01-21 01:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("plan", "0032_set_plan_queues"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="plan",
+            name="build_timeout",
+            field=models.IntegerField(default=28800),
+        ),
+        migrations.AlterField(
```

Comment:
@prescod - Do you know why this migration says it wants to alter the `Plan.queue` field?

Reply:
I would have guessed that it should only be adding the new build_timeout field.

Reply:
Sorry I wasn't clear in the PR comment. Makemigrations created that because at some point in the past someone didn't run makemigrations after they changed a model. It can be a bit tricky to know what model changes will trigger a migration so it is easy to forget. 

We have discussed adding a validation in our build system for that, so you could add that to a todo list if you like.

There was actually another migration generated. I ignored it because it looked empty and wasn't in this file at all. I don't remember which model it was on but it looked empty.

Reply:
Also: the code coverage went up when I did my commits but seemed to go down when I merged master. So I think that its master that went down, not my PR.

Reply:
Ya I'm confused by the (-0.1%) in Coveralls (which I saw in my last MetaCI PR on Friday as well).

Reply:
Would it be ok to hold off on merging this until I deploy to the metaci robot app tonight? (I want to ensure that both apps are on the same version and don't want to have to deploy to both tonight).